lib/block.js

```diff
@@ -6,74 +6,75 @@
  * See the included LICENSE file for more details.
  */
 
-function isNullOrUndefined(obj) {
-  return typeof obj === "undefined" || obj === null
+function isNullOrUndefined (obj) {
+    return typeof obj === 'undefined' || obj === null
 }
-var TwoPowTwenty = 1048575
+const TwoPowTwenty = 1048575
 /**
- * 
+ *
  * @param {Number} sequenceNumber The block sequence number.
  * @param {Boolean} moreBlocks If there are more blocks to follow
- * @param {Number} blockSize 
+ * @param {Number} blockSize
  */
-module.exports.generateBlockOption = function(sequenceNumber, moreBlocks, blockSize) {
-  if(typeof sequenceNumber === "object") {
-    moreBlocks = sequenceNumber.moreBlocks
-    blockSize = sequenceNumber.blockSize
-    sequenceNumber = sequenceNumber.sequenceNumber
-  }
+module.exports.generateBlockOption = function (sequenceNumber, moreBlocks, blockSize) {
+    if (typeof sequenceNumber === 'object') {
+        moreBlocks = sequenceNumber.moreBlocks
+        blockSize = sequenceNumber.blockSize
+        sequenceNumber = sequenceNumber.sequenceNumber
+    }
+
+    if (isNullOrUndefined(sequenceNumber) || isNullOrUndefined(blockSize) || isNullOrUndefined(sequenceNumber)) {
```

Comment:
I just noticed that this line can be refactored like so:

```suggestion
    if (sequenceNumber == Null || blockSize == Null || sequenceNumber == Null {
```

This change would make the `isNullOrUndefined` function obsolete.

Reply:
But the change could lead with some eslink checks to "== is bad because types can change blaaahh" warnings .. so we should suppress them? ;) but agreed that just because if this having a method makes not that much sense

Reply:
The check for ´null` and `undefined` is actually the one expection where can use `==` instead of `===` according to their [rules](https://standardjs.com/rules.html) (see `eqeqeq`). So I think it should be safe to use it without suppressing them :)

I noticed, though, that I made three typos in the original suggestion (`Null` instead of `null`🤦‍♂️). Those are fixed now, however :)

Reply:
Yes also wanted to mentionthe capital "N"s, but lets do it that way.